src/components/AppContext.ts

```diff
@@ -0,0 +1,12 @@
+import {createContext} from 'react';
+
+export interface AppContextValue {
```

Comment:
```suggestion
export interface AppContextProps {
```

Reply:
Done in https://github.com/vadimdemedes/ink/pull/264/commits/ce79b7560a3ce9c6aff96778caae9c037b583af5